client/app/components/Input/InputPassword.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React from 'react';
+import css from './InputPassword.scss';
+
+export type Props = {
+  text?: string,
+  show?: boolean,
+  onChange?: Function,
+  onClick?: Function,
+};
+
+export type State = {
+  show?: boolean,
+};
+
+export class InputPassword extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { show: props.show };
+  }
+
+  handleOnChange = (
+    event: SyntheticEvent<HTMLInputElement>,
+    onChange?: Function,
+    text?: string,
+  ) => {
+    text = event.currentTarget.value;
+    if (onChange) {
+      onChange({ text });
+    }
+  };
+
+  toggleShow = (
+    event: SyntheticEvent<HTMLButtonElement>,
+    onClick?: Function,
+    show?: boolean,
+  ) => {
+    (event.currentTarget: HTMLButtonElement);
+    this.setState({ show: !show });
+  };
+
+  render() {
+    const {
+      id, show, text, onChange, onClick
+    } = this.props;
+    return (
+      <div>
+        <input
+          type = {this.state.show ? "text" : "password"}
```

Comment:
Let's remove the spacing between the equal signs throughout this file.

e.g. `type={this.state.show ? "text" : "password"}`